src/main/java/com/expediagroup/hiveberg/IcebergSchemaToTypeInfo.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (C) 2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.hiveberg;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.hadoop.hive.serde.serdeConstants;
+import org.apache.hadoop.hive.serde2.typeinfo.HiveDecimalUtils;
+import org.apache.hadoop.hive.serde2.typeinfo.TypeInfo;
+import org.apache.hadoop.hive.serde2.typeinfo.TypeInfoFactory;
+import org.apache.iceberg.Schema;
+import org.apache.iceberg.types.Type;
+import org.apache.iceberg.types.Types;
+
+import com.google.common.collect.ImmutableMap;
+
+/**
+ * Class to convert Iceberg types to Hive TypeInfo
+ */
+class IcebergSchemaToTypeInfo {
```

Comment:
This looks more like a util class at the moment that doesn't have any non-static state. So you could consider making it final, adding an empty private constructor and making `getColumnTypes` static.

Reply:
I agree, I think it makes sense to make it into a util class